### DISCUSSION TITLE
Rest API call settings cleared issue

### DISCUSSION THREAD

**Discussion by valepetros** [LiveHelperChat/livehelperchat#2138] [Q&A]
Hello,

In the Rest API Calls section I've configured correctly all the params using a json request to chatgpt.
If I try to write a very long text I lose all the rest api configuration.

Is it possible that the field on the database has a limited amount of characters? Can I increase it?
Or is a problem of special characters?

thanks

**Comment by remdex** [comment]
After save your text should be saved in db in any case id you export that record i could try myself. I ques you somehow get malformed json in some part. Without more details can't help...

**Comment by valepetros** [comment]
Te json body is this:

`{
    "model": "gpt-3.5-turbo",
    "messages": [
      {
        "role": "system",
        "content": "Sei l'assistente digitale..."
      },
      {
        "role": "user",
        "content": {{msg_url}}
      }
    ]
  }`

**Reply by remdex** [reply]
I'll test tomorrow and let you know :)

**Comment by remdex** [comment]
Sorry,
But I don't have this problem... Everything saves on my env. You will have to find out what breaks on your env :)

**Reply by remdex** [reply]
P.s you can try to define/import your call on official demo if you can break it then I look :)
